README.md

```diff
@@ -150,11 +156,10 @@ npm run-script coverage
 - [merkle-lib](https://github.com/bitcoinjs/merkle-lib) - A performance conscious library for merkle root and tree calculations.
 - [minimaldata](https://github.com/bitcoinjs/minimaldata) - A module to check bitcoin policy: SCRIPT_VERIFY_MINIMALDATA
 
-
 ## Alternatives
+
```

Comment:
The following libraries are no longer compatible with Tapayrus and should be removed.

Reply:
1a6db341811f1715df5547386e535877ee010ff3